html/pfappserver/lib/pfappserver/PacketFence/Controller/Interface.pm

```diff
@@ -355,6 +360,24 @@ around create_action => sub {
     return $action;
 };
 
+=item audit_current_action
+
+Create an audit log entry
+
+=cut
+
+sub audit_current_action {
```

Comment:
Yes as this controller does not inherit from pfappserver::Base::Controller

However it does make more sense to create it as a role